Updated featurizers (#4935)
RasaHQ/rasa: changelog/4957.removal.rst

> To use custom features in the ``CRFEntityExtractor`` use ``text_dense_features`` instead of ``ner_features``. If

Reviewer: ```suggestion
Removed ``ner_features`` as a feature name from ``CRFEntityExtractor ``, use ``text_dense_features `` instead. If
```

Reviewer: please also add an empty line after the first sentence.
